src/finalizer/utils/opStack.ts

```diff
@@ -129,7 +130,7 @@ export async function opStackFinalizer(
     spokePoolClient.getTokensBridged().filter(
       ({ l2TokenAddress }) =>
         // CCTP USDC withdrawals should be finalized via the CCTP Finalizer.
-        !compareAddressesSimple(l2TokenAddress, USDC.addresses[chainId]) || !(getCctpDomainForChainId(chainId) > 0)
+        !l2TokenAddress.eq(toAddressType(USDC.addresses[chainId], chainId)) || !(getCctpDomainForChainId(chainId) > 0)
```

Comment:
This is totally EVM so we should probably use `EvmAddress.from()` here.

```suggestion
        !l2TokenAddress.eq(EvmAddress.from(USDC.addresses[chainId])) || !(getCctpDomainForChainId(chainId) > 0)
```